tendermint/src/lite/types.rs

```diff
@@ -180,12 +185,8 @@ where
 }
 
 /// SignedHeader bundles a [`Header`] and a [`Commit`] for convenience.
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)] // NOTE: Copy/Clone/Debug for convenience in testing ...
-pub struct SignedHeader<C, H>
-where
```

Comment:
I removed the bounds here to avoid having to use the `#[serde(bound = ...)]` trick, but also because it is [considered a best practice](https://github.com/rust-lang/rust-clippy/issues/1689) to not add bounds on structs when one can avoid it.